script.js

```diff
@@ -1 +1,25 @@
-console.log('this works');
\ No newline at end of file
+// Set 1
+
+var div;
+
+// create a buildBox function
+function buildBox() {
+	// create and insert an empty div on the page with: positioning, top and left coordinates, background color expressed as rgba 
+	div = document.createElement('div');
+	div.style.cssText = 'width: 300px; height: 300px; position: absolute; top: 0; left: 0; background-color: rgba(0,0,0,1);';
```

Comment:
I might be reading the article wrong, but it seems like I should be using a function declaration and not a function expression since function declarations will be hoisted to the top of the JS scope? That's what I'm currently doing.